robot/Cumulus/tests/browser/donations_payments/lead_conversion.robot

```diff
@@ -1,49 +1,42 @@
 *** Settings ***
-
+Library  String
 Resource        robot/Cumulus/resources/NPSP.robot
 Library         cumulusci.robotframework.PageObjects
 ...             robot/Cumulus/resources/ContactPageObject.py
 ...             robot/Cumulus/resources/AccountPageObject.py
+...             robot/Cumulus/resources/LeadsPageObject.py
+
 Suite Setup     Open Test Browser
 Suite Teardown  Delete Records and Close Browser
 
-
 *** Test Cases ***
 
 Convert Lead To Account
     ${first_name} =           Generate Random String
     ${last_name} =            Generate Random String
-    ${company} =              Generate Random String  
-    Go To Object Home         Lead
-    Click Object Button       New
-    Populate Form
-    ...                       First Name=${first_name}
-    ...                       Last Name=${last_name}
-    ...                       Company=${company}
-    Search Field By Value          Street            50 Fremont Street  
-    Search Field By Value          City              San Francisco
-    Search Field By Value          Zip/Postal Code   95320
-    Search Field By Value          State/Province    CA
-    Search Field By Value          Country           USA
-    Select Value From Dropdown   Lead Status              Working - Contacted
-    Click Modal Button        Save    
+    ${company} =              Generate Random String
+    Go To Page                       Listing                 Lead
+    Click Object Button              New
+    populate modal form
```

Comment:
can u change to camel case for consistency

Reply:
Done